### PR TITLE
feat: improve update button visibility

### LINKED ISSUE #3024: Make workspace "Update" button stand alone
A recent change (#2748) combined the different workspace buttons to free up some space.  This is a good change, but there was a little collateral damage.  I think the "Update" button shouldn't be grouped; if the template is out of date, it should always be top level instead of in the collapsed array.  OR "Update" could be the first button with "Start" being in the drop-down.

Definition of Done:
Update call to action should be more prominent. 

image

AC as per below discussion:
 - keep `Update` in button dropdown
 - make it the primary CTA when an update is available
 - add an 'Outdated' tooltip (similar to the one in the workspaces table) when an update is available

### PR DESCRIPTION
**Pull request by Kira-Pilot**
resolves #3024

To improve visibility, the `Update` button is always given primary focus in the workspace actions dropdown, if applicable.

In addition, we've added the `Update` tooltip that lives in the workspaces table to the workspace view.
Screen Shot 2022-07-21 at 5 09 57 PM

### CODE REVIEW

File: site/src/components/WorkspaceActions/WorkspaceActions.tsx [hunk 1]
```diff
@@ -34,7 +41,23 @@ export const WorkspaceActions: FC<WorkspaceActionsProps> = ({
     workspace.latest_build,
   )
   const workspaceState = WorkspaceStateEnum[workspaceStatus]
-  const actions = WorkspaceStateActions[workspaceState]
+
+  const canUpdateWorkspace = workspace.outdated && canAcceptJobs(workspaceStatus)
```
Comment (AbhineetJain) [inline site/src/components/WorkspaceActions/WorkspaceActions.tsx:45]: `canUpdateWorkspace` has been used previously to refer to update permissions on the workspace: https://github.com/coder/coder/blob/main/site/src/pages/WorkspacePage/WorkspacePage.tsx#L37. Maybe we can rename it to `canBeUpdated`?

File: site/src/components/WorkspaceStats/createDayString.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import dayjs from "dayjs"
+
+/**
+ * Returns a human-readable string describing when the workspace was created
+ * Broken into its own module for testing purposes
+ */
+export function createDayString(createdAt: string): string {
```
Comment (AbhineetJain) [inline site/src/components/WorkspaceStats/createDayString.ts:7]: Should we move this to a `util` and update other instances to use this method? https://github.com/coder/coder/search?q=dayjs%28%29.to

**Review by AbhineetJain — APPROVED** [review]

**Comment by AbhineetJain** [comment]
Great change! 🙌

File: site/src/components/WorkspaceStats/createDayString.ts (hunk 2 above)
Comment (Kira-Pilot) [reply site/src/components/WorkspaceStats/createDayString.ts:7]: gooooood thinking

**Pull request merged**